.github/workflows/tests.yml

```diff
@@ -130,6 +130,8 @@ jobs:
           sudo snap set lxd daemon.group=adm
           sudo lxd init --auto
           snap list
+          sudo apt install -y linux-modules-extra-$(uname -r)
+          sudo modprobe vrf
```

Comment:
We have a precedent in the code base for installing kernel modules through LXD:
https://github.com/canonical/microovn/blob/31ce8c71de4f3ab6074d71ad74579a2d73da31a2/tests/lxd-instance-config.yaml#L3

Using that would make loading automatic regardless of environment, and presumably it would also provide a useful error message if the module is not available?

Reply:
Excellent point! thank you. I didn't realize that we can use this mechanism. The `lxc launch` fails with
```
Error: Failed to load kernel module 'vrf': Failed to run: modprobe -b vrf: exit status 1
```
if the module is not available.

Reply:
I had bit of a dilemma here as I didn't want to enforce `vrf` kernel module for all containers and I didn't want to duplicate the profile either. I ended up using `launch_containers_args` function that allows us to pass `-c linux.kernel_modules=vrf,openvswitch` arg that overrides the profile just for the containers that need the vrf.

My concern with requiring VRF for all containers was that it' only needed in single test suite and reuqires installation of quite heavy package `linux-modules-extra`. Given that we distribute our test runs across many machines, most of them would install this package unnecessarily.